src/select-list-view.js

```diff
@@ -37,6 +37,14 @@ module.exports = class SelectListView {
     this.refs.queryEditor.setText('')
   }
 
+  setQueryText (value, isSelect = false) {
```

Comment:
What do you think about renaming this method to `setQuery` and the parameter to `selectAll`?

Also, I think we should make the second parameter an object, so that it's clear what the boolean stands for, e.g.

```js
setQuery (text, {selectAll} = {})
```